app/posts/apply-for-teacher-training/2023-06-21-exploring-ways-to-integrate-the-services.md

```diff
@@ -0,0 +1,138 @@
+
+The messages feature could also perhaps include automated notifications, such as when an offer is received from a provider.
+
+## Criminal records checks
+
+Once a candidate has accepted an offer for a training course, they are required to pass safeguarding checks including an enhanced criminal records check. This is currently all done off-service, with providers asked to manually mark the status of the check as ‘complete’ once it has been done.
+
+To make this more seamless, the service could give candidates a link to apply for a criminal records check, and then ask for their ‘share code’ once the check has been done.
+
+![Screenshot showing a page asking for a criminal records check share code](criminal-records-check.png)
+
+![Screenshot showing a page where criminal records checks have been successful](criminal-records-check-done.png)
+
+This would require tight integration with the [Disclosure and Barring Service (DBS)](https://www.gov.uk/government/organisations/disclosure-and-barring-service). Currently they don’t have an online service for enhanced checks, only basic ones, but this may change in future.
+
+## Scholarships
+
+Candidates who are offered a place on a course for physics, maths, chemistry, computing or modern languages are offered the ability for apply for a scholarship. These are administered by professional bodies such as the Institute of Physics, and offer slightly more money than the standard bursary, as well as other benefits.
+
+However applying for these currently involves filling out entirely separate application forms (asking for much of the same information as already entered on Apply), and also candidates have to know that this is available to them.
+
+Instead of this, the service could prompt all eligible candidates to apply, and could make this a single click rather than asking for any extra information.
+
+![Screenshot showing a page inviting candidates to apply for a scholarship](scholarship.png)
+
+![Screenshot showing a page confirming that a scholarship application has been sent](scholarship-done.png)
+
+This would involve working closely with the scholarship institutions and providing them a way to receive and make decisions on scholarship applications.
+
+## Student loan
+
+Most candidates applying to fee-paying courses will go on to apply for a student loan in order to pay for the course fee as well as contribute to their living costs.
+
+Currently this involves following the guidance and application process for [undergraduate student finance](https://www.gov.uk/student-finance) (even though the course is post graduate), which involves adding details of their course as well as proving that they have met the conditions of their offer.
+
+This could be improved through better integration with the Student Finance England, which might enable to apply for a student loan without having to re-enter and prove the details of their training course.
```

Comment:
```suggestion
This could be improved through better integration with the Student Finance England, which might enable candidates to apply for a student loan without having to re-enter and prove the details of their training course.
```